tsd/src/tsd/authoring/importers.hpp

```diff
@@ -20,6 +20,9 @@ void import_PLY(Context &ctx, const char *filename, LayerNodeRef location = {});
 void import_PDB(Context &ctx, const char *filename, LayerNodeRef location = {});
 void import_SWC(Context &ctx, const char *filename, LayerNodeRef location = {});
 void import_XYZDP(Context &ctx, const char *filename, LayerNodeRef location = {});
+#ifdef USE_NEURAL_GRAPHICS_PRIMITIVES
```

Comment:
I tend to prefer the pattern that the import functions are always present, but throw an error in the log when invoked and not enabled so there is feedback to end-users that a TSD app _can_ import something, it just has to be built with the support enabled. See others as an example (such as `import_ASSIMP.cpp`).